### PR TITLE
Team6

### PR DESCRIPTION
**Pull request by tanyihao**
request pull master, no conflicts

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,7 @@ TestResults
 *.suo
 *.user
 *.sln.docstates
+Startup.cs
```
Comment (JohnLeong) [inline .gitignore]: why is Startup.cs being ignored?

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,63 @@
+###############################################################################
```
Comment (JohnLeong) [inline .gitattributes]: what is this file for?

File: HotelManagementSystem-Module1/HotelManagementSystem-Module1/HotelManagementSystem.csproj [hunk 3]
```diff
@@ -5,6 +5,17 @@
     <RootNamespace>HotelManagementSystem</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```
Comment (JohnLeong) [inline HotelManagementSystem-Module1/HotelManagementSystem-Module1/HotelManagementSystem.csproj]: what is this? (the next few lines also)

File: .gitignore (hunk 1 above)
Comment (tanyihao) [reply .gitignore]: removed from ignore

File: .gitattributes (hunk 2 above)
Comment (tanyihao) [reply .gitattributes]: deleted

File: HotelManagementSystem-Module1/HotelManagementSystem-Module1/HotelManagementSystem.csproj (hunk 3 above)
Comment (tanyihao) [reply HotelManagementSystem-Module1/HotelManagementSystem-Module1/HotelManagementSystem.csproj]: deleted

**Review by JohnLeong — COMMENTED** [review]
delete all DS_store files or add it to the gitignore

**Review by JohnLeong — APPROVED** [review]

**Pull request merged**